### DISCUSSION TITLE
Compatibility with Gradle 7.5+

### DISCUSSION THREAD

**Discussion by dmurat** [kordamp/kordamp-gradle-plugins#519] [Q&A] [answered]
I had an impression that kordamp-gradle-plugins 0.49 works with Gradle 7.5. Since this is not the case, can you please, provide some details about Gradle 7.5+ support? Also, when is it expected to be available, if at all? Since I need at least 7.5 support, I'm considering available options and if I have to move away from kordamp-gradle-plugins. Thank you.

**Comment by aalmiray** [comment, marked as answer]
I'm using `0.49.0` for kordamp/jarviz and so far I have not experienced any issues with it. However, I've fixed most of the incompatibilities reported so far, as well as removed plugins that are no longer needed. That's what's coming in `0.50.0`.

Do you have any other issues open with `0.49.0`?

**Comment by dmurat** [comment]
I see. I wasn't sure if it was intended to work with 7.5+ or not. Now that I know, I will report the issue then. Thanks for the explanation.
